[2023-06-01T11:21:15.684+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-03-06T00:00:00+00:00 [queued]>
[2023-06-01T11:21:15.737+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-03-06T00:00:00+00:00 [queued]>
[2023-06-01T11:21:15.738+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:21:15.739+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:21:15.741+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:21:15.811+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-03-06 00:00:00+00:00
[2023-06-01T11:21:15.841+0000] {standard_task_runner.py:55} INFO - Started process 1088 to run task
[2023-06-01T11:21:15.896+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-03-06T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpo4wste82']
[2023-06-01T11:21:15.900+0000] {standard_task_runner.py:83} INFO - Job 285: Subtask insert_into_table
[2023-06-01T11:21:16.264+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-03-06T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:21:16.766+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=insert_into_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-06T00:00:00+00:00
[2023-06-01T11:21:16.769+0000] {sql.py:253} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-03-06', 'dag_with_postgres_operator_v03')
        
[2023-06-01T11:21:16.803+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:21:18.506+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:21:18.529+0000] {sql.py:364} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-03-06', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-06-01T11:21:18.539+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-01T11:21:18.578+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20220306T000000, start_date=20230601T112115, end_date=20230601T112118
[2023-06-01T11:21:18.725+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:21:18.803+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
